src/pages/index.ejs

```diff
@@ -987,7 +987,7 @@ function activeLink(urlPart) {
             </div>
             <div class="call-to-action-content-action">
               <div class="call-to-action-content-action-copy-command">
-                <p class="call-to-action-content-action-copy-command-text">npm init nightwatch</p>
+                <p class="call-to-action-content-action-copy-command-text"><abbr title="npm init nightwatch">npm init nightwatch</abbr></p>
```

Comment:
`npm init nightwatch` isn't really an abbreviation to use the [`<abbr>` HTML element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/abbr), you could just do the following:
```suggestion
                <p class="call-to-action-content-action-copy-command-text" title="npm init nightwatch">npm init nightwatch</p>
```

Reply:
ok , i done this also @garg3133